### DISCUSSION TITLE
can';t delete my self from suppression list (reason: too many soft fails)

### DISCUSSION THREAD

**Discussion by moyoussry** [postalserver/postal#2619] [Help with using Postal]
The recipient (my email @gmail.com) is on the suppression list (reason: too many soft fails)

I have installed postal server with docker and I want to edit the database or even any way to delete myself from this list but I can't

**Comment by willpower232** [comment]
The fastest way to remove an entry from the suppression list is to release a message to that email address and have it successfully get there.

According to the code, the suppression list for each server within Postal is kept in the message db which means you need to track down which postal-whatever database relates to the one you're looking at and then you should see the suppressions table.

https://github.com/postalserver/postal/blob/main/lib/postal/message_db/migrations/14_create_suppressions.rb
